packages/web/components/Player/ColorBar.tsx

```diff
@@ -57,25 +65,34 @@ export const ColorBar = ({
           );
         })}
       </Flex>
-      <Flex minH="calc(1.5rem + 4px)" maxW="100%" border="2px" borderRadius={3}>
+      <Flex
+        minH="calc(1.5rem)"
+        flex="0 0 100%"
+        minW="100%"
+        border="0"
+        borderRadius="15px"
+        overflow="hidden"
+      >
         {parts.map((part) =>
           (mask & part.mask) === 0 ? null : (
-            <Flex key={part.mask} grow={1} h="1.5rem">
-              <SVG viewBox="0 0 100 100" preserveAspectRatio="none" w="100%">
-                <defs>
-                  <linearGradient id="shading" gradientTransform="rotate(90)">
-                    <stop offset="5%" stopColor="black" stopOpacity={0.5} />
-                    <stop offset="95%" stopColor="white" stopOpacity={0.25} />
-                  </linearGradient>
-                </defs>
-                <rect width="100%" height="100%" fill={colors[part.mask]} />
-                <rect width="100%" height="100%" fill="url(#shading)" />
-              </SVG>
-            </Flex>
+            <Flex
+              key={part.mask}
+              grow={1}
+              h="1.5rem"
+              bg={`linear-gradient(to right, ${colors[part.mask].start}, ${
+                colors[part.mask].end
+              })`}
+            />
           ),
         )}
       </Flex>
-      <FlexContainer mt={1}>
+      <FlexContainer
+        mb={2}
+        fontFamily="mono"
+        sx={{
+          q: { '&::before': { content: "''" }, '&::after': { content: "''" } },
```

Comment:
It seems like if you're removing the quote characters from the `q` element, you could just use a `Box` instead.

Reply:
I have replaced with a `<Box as="span">`